docs/src/increase-observability/metrics/dedicated.md

```diff
@@ -0,0 +1,103 @@
+| :------------------------ | :------------------------------------------------------------------ |
+| `Host i-04353f1e6f`       | `ssh 3.ent-abcde3clusterID-production-qwerty8@ssh.us-4.platform.sh` |
+| `Host i-04d1ac8319`       | `ssh 2.ent-abcde3clusterID-production-qwerty8@ssh.us-4.platform.sh` |
+| `Host i-0b1e1b96cf`       | `ssh 1.ent-abcde3clusterID-production-qwerty8@ssh.us-4.platform.sh` |
+
+To get the host ID from an SSH connection, SSH into the host:
+
+```bash
+$ ssh 3.ent-abcde3clusterID-production-qwerty8@ssh.us-4.platform.sh
+
+ ___ _      _    __                    _
+| _ \ |__ _| |_ / _|___ _ _ _ __    __| |_
+|  _/ / _` |  _|  _/ _ \ '_| '  \ _(_-< ' \
+|_| |_\__,_|\__|_| \___/_| |_|_|_(_)__/_||_|
+
+
+ Welcome to Platform.sh.
+
+ This is environment production-qwerty8 of project abcde3clusterID.
+
+abcde3clusterID@i-04d1ac8319f6ab9a6:~$ 
+```
+
+The relevant string for the host ID is shown after the `@` and before the 7-character string at the end (`f6ab9a6`).
+In this case, the ID is: `i-04d1ac8319`.
+You can then match this ID with the one under **Metrics** for your investigations.
+
+## Multiple clusters
+
+Standard Dedicated environments have a single cluster of three hosts,
+where each additional cluster adds at least three additional hosts to the project.
+Dedicated environments with two clusters display metrics in two separate groups:
+one for the web hosts that handle web requests and one for the service hosts that handle database, cache, and other services.
+
+![Metrics split into Web hosts and Service hosts](/images/metrics/split-arch.png "0.75")
+
+## Example of how to read metrics
+
+This example should give you an idea of how the metrics appear.
+Dedicated environments metrics show the resource usage per host within the cluster.
+
+This reference project has configured a single PHP application and three services: MySQL, Redis, and Solr.
+It has been granted 4&nbsp;GB of memory and 2 vCPUs per host based on its plan.
+
+![All of the metrics available for a Dedicated environment](/images/metrics/all.png "0.5")
+
+### CPU
+
+The project has 2 virtual CPUs (vCPUs) allocated per host, as seen in the upper limit for the CPU metric.
+
+![A closeup of the CPU metrics](/images/metrics/cpu.png "0.5")
+
+### RAM
+
+RAM measures overall RAM usage, displaying the percentage of the RAM available for each host.
+
+For this example, each host has been granted 4&nbsp;GB of memory.
+The upper limit in the graph is 3.62&nbsp;GB because 0.38&nbsp;GB of memory are in use by the operating system.
+
+![A closeup of the RAM metrics](/images/metrics/ram.png "0.5")
+
+### Disk
+
+Disk represents the plan space for all services and mounts,
+which for Dedicated production environments is [50&nbsp;GB](../../dedicated/architecture/_index.md).
+
+This example has the following persistent disk space configured:
+
+* App: 9.78&nbsp;GB for the mount `exports`
+* MySQL: 9.78&nbsp;GB
+* Redis: no persistent disk
+
```

Comment:
🛠️ To avoid unclarity, we could add the detail on how much space is left for further growth?
Otherwise, it feels like "I have 50 GB", "X GB are used for the apps... but what about the rest?"

Reply:
I'm not sure what insight would be gained by that. The metrics themselves show the available space remaining. This list should clarify the context for what can be found in the metrics themselves.